DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf

```diff
@@ -6973,11 +6912,42 @@
           "Id": "e839a46c-db69-49ad-abea-698e8ba258be",
           "LastModified": "2020-10-29T15:48:42.667Z"
         },
+        {
+          "ParentTypeId": "63177677-3adb-4cb7-8bd2-136e4c34201f",
+          "FieldNamespace": "Telerik.Sitefinity.DynamicTypes.Model.JobProfile.Skill",
+          "Name": "PSFHidden",
+          "Title": "    PSFHidden",
+          "FieldType": 4,
+          "SpecialType": 0,
+          "MediaType": "",
+          "FieldTypeDisplayName": "YesNo",
+          "LengthValidationMessage": "",
+          "Ordinal": 6,
+          "CanCreateItemsWhileSelecting": true,
+          "WidgetTypeName": "Telerik.Sitefinity.Web.UI.Fields.ChoiceField",
+          "DBType": "BIT",
+          "ClassificationId": "cb0f3a19-a211-48a7-88ec-77495c0f5374",
+          "DBLength": "",
+          "InstructionalChoice": "- Select -",
+          "NumberUnit": "",
+          "ColumnName": "",
+          "InstructionalText": "",
+          "ImageExtensions": "",
+          "VideoExtensions": "",
+          "FileExtensions": "",
+          "DefaultValue": "",
+          "Choices": "",
+          "ChoiceRenderType": "",
+          "ParentSectionId": "136991b1-e317-4cec-9cce-6bc53fc6db84",
+          "TypeUIName": "\n                                Yes / No\n                            ",
+          "Id": "e89d7654-2e60-4539-80cc-bebe5d557f8d",
+          "LastModified": "2020-10-30T09:16:27.243Z"
+        },
         {
           "ParentTypeId": "63177677-3adb-4cb7-8bd2-136e4c34201f",
           "FieldNamespace": "Telerik.Sitefinity.DynamicTypes.Model.JobProfile.Skill",
           "Name": "PSFNotApplicable",
-          "Title": "PSFNotApplicable",
+          "Title": "    PSFNotApplicable",
```

Comment:
This intentional?

Reply:
yep just to add a gap between title and the checkbox. Doesn't effect the code